linked_list/linkedList.c.c

```diff
@@ -0,0 +1,255 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node {
+	int d;			//integer type data
+	struct Node* next;	// pointer to same structure
+};
+
+//  Creating node
+struct Node* getNewNode(int data) {
+	struct Node *nw_node;
+	nw_node = (struct Node*) malloc(sizeof(struct Node));	//  nw_node a Node to be added
+	if (nw_node == NULL) {
+		return NULL;
+	}
+	nw_node->d = data;
+	nw_node->next = NULL;
+	return nw_node;
+}
+
+//  Adding at Front
+int addFront(struct Node **start, struct Node* nw_node) {
```

Comment:
@sakshee-19: No, just call `addFront`, `addLast`, etc. in these functions - 

```c
int addFront(struct Node **start, struct Node* nw_node) {
    return addAtPos(start, nw_node, 0);
}
```

and similarly for others.